services/brig/test/integration/API/Calling.hs

```diff
@@ -15,37 +17,46 @@
 -- You should have received a copy of the GNU Affero General Public License along
 -- with this program. If not, see <https://www.gnu.org/licenses/>.
 
-module API.TURN where
+module API.Calling where
 
 import Bilge
 import Bilge.Assert
+import qualified Brig.Options as Opts
 import Brig.Types
-import Control.Lens ((^.))
+import Control.Lens ((?~), (^.), view)
+import Control.Monad.Catch (MonadCatch, MonadThrow)
+import Data.Bifunctor (Bifunctor (first))
 import Data.ByteString.Conversion
 import qualified Data.ByteString.Lazy as LB
 import Data.Id
 import Data.List ((\\))
+import qualified Data.List.NonEmpty as NonEmpty
 import Data.List1 (List1)
 import qualified Data.List1 as List1
-import Data.Misc (Port)
+import Data.Misc (Port, mkHttpsUrl)
+import qualified Data.Set as Set
 import Imports
 import Network.HTTP.Client (Manager)
 import System.FilePath ((</>))
 import Test.Tasty
 import Test.Tasty.HUnit
+import URI.ByteString (laxURIParserOptions, parseURI)
 import UnliftIO.Exception (finally)
 import qualified UnliftIO.Temporary as Temp
 import Util
-
-tests :: Manager -> Brig -> FilePath -> FilePath -> IO TestTree
-tests m b turn turnV2 = do
-  return $
-    testGroup
-      "turn"
-      [ test m "basic /calls/config - 200" $ testCallsConfig b,
-        -- FIXME: requires tests to run on same host as brig
-        test m "multiple servers /calls/config - 200" . withTurnFile turn $ testCallsConfigMultiple b,
-        test m "multiple servers /calls/config/v2 - 200" . withTurnFile turnV2 $ testCallsConfigMultipleV2 b
+import Wire.API.Call.Config
+
+tests :: Manager -> Brig -> Opts.Opts -> FilePath -> FilePath -> IO TestTree
+tests m b opts turn turnV2 = do
```

Comment:
I think this is because in some integration tests we do some IO things for all the `testGroups` and here we just kept it similar to that. I think it is better have the `tests` functions be an IO everywhere rather than in only some places. But I don't really feel very strongly on this one.